src/net/download.rs

```diff
@@ -1797,18 +1794,22 @@ impl PeerNetwork {
                     }
                 }
 
-                if downloader.blocks_to_try.len() == 0 || downloader.microblocks_to_try.len() == 0 {
-                    // nothing in this range, so advance sortition range to try for next time
-                    if downloader.blocks_to_try.len() == 0 {
-                        next_block_sortition_height = next_block_sortition_height
-                            + (network.burnchain.pox_constants.reward_cycle_length as u64);
-                    }
-                    if downloader.microblocks_to_try.len() == 0 {
-                        next_microblock_sortition_height = next_microblock_sortition_height
-                            + (network.burnchain.pox_constants.reward_cycle_length as u64);
-                    }
-
-                    debug!("{:?}: Pessimistically increase block and microblock sortition heights to ({},{})", &network.local_peer, next_block_sortition_height, next_microblock_sortition_height);
+                // nothing in this range, so advance sortition range to try for next time
```

Comment:
Technically, this comment should go under both of these `if`-statements, since the logic here is meant to advance the pointers into the chainstate at which to begin the next scan for available blocks and microblock streams.